### PR TITLE
fixing the fields name

### PR DESCRIPTION
**Pull request by biozit**

### CODE REVIEW

File: Collectors/DetailedCollector.py [hunk 1]
```diff
@@ -295,8 +295,11 @@ def process_gstream(self, gstream, addr, sid):
                      event["server_hostname"] = hostname
                      event["file_path"] = event.pop("lfn")
                      event["block_size"] = event.pop("blk_size")
+                     # number of blocks used by the file
                      event["numbers_blocks"] = event.pop("n_blks")
+                     # number of blocks read by the cache
```
Comment (djw8605) [inline Collectors/DetailedCollector.py:300]:
```suggestion
                     # number of blocks that have been downloaded by the cache
```

File: Collectors/DetailedCollector.py [hunk 2]
```diff
@@ -295,8 +295,11 @@ def process_gstream(self, gstream, addr, sid):
                      event["server_hostname"] = hostname
                      event["file_path"] = event.pop("lfn")
                      event["block_size"] = event.pop("blk_size")
+                     # number of blocks used by the file
```
Comment (djw8605) [inline Collectors/DetailedCollector.py:298]:
```suggestion
                     # Total number of blocks in the file
```

**Review by djw8605 — CHANGES_REQUESTED** [review]
A few changes to clarify the fields

**Comment by biozit** [comment]
> A few changes to clarify the fields

ok

**Review by djw8605 — APPROVED** [review]
LGTM

**Pull request merged**